WcaOnRails/app/models/competition.rb

```diff
@@ -979,6 +982,23 @@ def announced?
     announced_at.present? && announced_by.present?
   end
 
+  def cancelled?
+    cancelled_at.present? && cancelled_by.present?
+  end
+
+  def can_be_cancelled?
+    confirmed? && announced? && !cancelled?
+  end
+
+  def display_name(display_cell_name = false)
```

Comment:
Nit: it would make sense to use keyword argument here, `comp.display_name(true)` is not intuitive. I'm thinking like `comp.display_name(short: true)`.

Reply:
Good point and better naming, thanks!